tools/realsense-viewer/realsense-viewer.cpp

```diff
@@ -194,6 +194,8 @@ void refresh_devices(std::mutex& m,
 
 int main(int argv, const char** argc) try
 {
+    rs2::log_to_console(RS2_LOG_SEVERITY_WARN);
```

Comment:
Is it by design?

Reply:
It is convenient. This does nothing when building on Windows (unless you bring back the console in CMake), but adds information on Linux.